doc/getting_started/installation_mac.rst

```diff
@@ -64,6 +64,10 @@ Install tools to build Zephyr binaries:
    cd ~/zephyr   # or to the folder where you cloned the zephyr repo
    pip3 install --user -r scripts/requirements.txt
 
+To install pyocd, a python 2 only  module, use::
```

Comment:
Same here:

    If you require pyocd, an open source python2 library for programming and
    debugging ARM Cortex-M microcontrollers, use this command::